recipe/recipe.yaml

```diff
@@ -37,71 +32,95 @@ source:
         - fix-argminmax.patch
 
 build:
-  number: 0
-  skip: not (match(python, python_min ~ ".*") and is_abi3)
-  python:
-    version_independent: true
+  number: ${{ build }}
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
```

Comment:
```suggestion
```

i think you can just skip this? or is this then executing `build.sh`?

Reply:
> or is this then executing build.sh?

exactly